changelogs/fragments/async_really_true.yml

```diff
@@ -0,0 +1,4 @@
+bugfixes:
+- async_status, module - changed to use real booleans as returns instead of 1 and 0 ints as booleans.
+- finished, test plugin - switched to use actual booleans for comparisson from using 1 and 0 ints as booleans.
+- started, test plugin -  swtiched to use actual booleans for comparisson from using 1 and 0 ints as booleans.
```

Comment:
The finished and started test plugins aren't updated in this PR, only their documentation, so I don't think there's anything worth mentioning here about them.

```suggestion
```